src/zos_files/zowe/zos_files_for_zowe_sdk/uss.py

```diff
@@ -145,15 +173,42 @@ def get_content_streamed(self, file_path, binary=False):
         response = self.request_handler.perform_request("GET", custom_args, stream=True)
         return response
 
-    def download(self, file_path, output_file, binary=False):
-        """Retrieve the contents of a USS file and saves it to a local file."""
+    def download(self, file_path: str, output_file: str, binary: bool = False):
+        """
+        Retrieve the contents of a USS file and saves it to a local file.
+
+        Parameters
+        ----------
+        file_path: str
+            Path of the file to be downloaded
+        output_file: str
+            Name of the file to be saved locally
+        binary: bool
+            Specifies whether the contents are binary
+        """
         response = self.get_content_streamed(file_path, binary)
         with open(output_file, "wb" if binary else "w", encoding="utf-8") as f:
             for chunk in response.iter_content(chunk_size=4096, decode_unicode=not binary):
                 f.write(chunk)
 
-    def upload(self, input_file, filepath_name, encoding=_ZOWE_FILES_DEFAULT_ENCODING):
-        """Upload contents of a given file and uploads it to UNIX file"""
+    def upload(self, input_file: str, filepath_name: str, encoding: str = _ZOWE_FILES_DEFAULT_ENCODING):
+        """
+        Upload contents of a given file and uploads it to UNIX file.
```

Comment:
Thoughts on updating this description for clarity?

```suggestion
        Uploads contents of a given file and saves it to a file at the given USS path.
```

Reply:
I agree with you. The revision is employed manually since numpy style in Pydoclint requires the first sentence to be imperative mood (No following 's' for verbs).